cosim/black-parrot-example/Makefile.design

```diff
@@ -88,13 +100,14 @@ NBF_NCPUS ?= 1
 
 VPATH := $(wildcard $(BLACKPARROT_SDK_DIR)/prog/*)
 %.nbf: %.riscv
+	cp $< $(CURR)
+	$(RISCV_OBJDUMP) -D $< > $*.dump
 	$(RISCV_OBJCOPY) -O verilog $< $*.mem
 	$(SED) -i "s/@8/@0/g" $*.mem
 	$(PYTHON) $(BP_COMMON_DIR)/software/py/nbf.py \
-		--debug --skip_zeros --config --boot_pc 0x80000000 --mem $*.mem --ncpus $(NBF_NCPUS) > $@
```

Comment:
I believe the current bootrom assumes debug and boot_pc = 0x100000. I see this isn't main, but probably want to sync up regular operation with the fuzzing